cumulusci/cumulusci.yml

```diff
@@ -1,7 +1,11 @@
 cumulusci:
     keychain: cumulusci.core.keychain.EncryptedFileProjectKeychain
-
+    
```

Comment:
Please don't add random whitespace.

Reply:
Im leaving as is since theres a space in master?

Reply:
@jofsky David's pointing at the extra four spaces on the blank line - you can see it show up in the diff for the PR (https://github.com/SFDO-Tooling/CumulusCI/pull/1387/files) or in `git diff`.

Reply:
There isn't in master; that's why it shows up in the diff. Master has a blank line with no spaces, but you added 4 spaces.